File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_pyramid(int height);

int main(void)
{
    const int MAX_HEIGHT = 8;
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > MAX_HEIGHT);
    print_pyramid(height);
}

void print_pyramid(int height)
{
    for (int i = 0; i < height; i++)
    {
        for (int j = 1; j <= height; j++)
        {
            if (j < height - i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }

        }
        printf("  ");
        for (int k = height - i; k <= height; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
import cs50


def main():
    while (True):
        height = cs50.get_int("Height: ")
        if (height > 0 and height < 9):
            break
    print_pyramid(height)


def print_pyramid(height):
    for i in range(0, height):
        for j in range(1, height + 1):
            if (j < height - i):
                print(" ", end="")
            else:
                print("#", end="")
        print("  ", end="")
        for k in range(height - i, height + 1):
            print("#", end="")
        print("")


if __name__ == "__main__":
    main()